### DISCUSSION TITLE
Add "Create Field" button at top of page

### DISCUSSION THREAD

**Discussion by codecatalysts** [directus/directus#14458] [Feature Requests]
Would be very beneficial and time saving to have a Create Field button at the top of the page not just sandwiched between the Field & Layout and Collection Setup sections. Collections with a high number of fields and groups the button gets lost and scrolled over as it blends with the form fields. A button at the top of the page much like on the "Create Collection" or in the Content.

<img width="854" alt="image" src="https://user-images.githubusercontent.com/19675191/179252386-a20d0f05-31b4-4780-bc01-d6d3bbb9b806.png">

<img width="854" alt="image" src="https://user-images.githubusercontent.com/19675191/179252423-8fa4f25c-28e6-4692-9d39-977d17c547f9.png">

**Comment by benhaynes** [comment]
To be clear, you are talking about adding the "Add Field" action button to the top of the **Settings > Data Model > [Collection]** page?

**Reply by codecatalysts** [reply]
Yes, that is correct. This option is in nearly every other screen that you can "create" something but not in the **Settings > Data Model > [Collection]** page.

**Reply by benhaynes** [reply]
I agree, and this seems like a straightforward addition. It's important to note that the **Add** button would trigger the "basic" flow (not advanced) though we could use the "..." icon beside it for the other options. @rijkvanzanten thoughts on all this?

We're focused on fixes/stability/tests right now, but if Rijk agrees, we could accept a properly considered PR for this. 👍

**Reply by rijkvanzanten** [reply]
I don't know if I agree. For the collection detail page, the primary action is to save the edits to the collection. To me, the fields are currently part of the edits made to the collection, rather than it's own primary action 🤔 The other benefit to the current position is that the field is added right where the "Add" button is (at the bottom of the existing fields). I agree that the scroll-reset is jarring, but that should be resolved by fixing the scrolling, rather than adding an additional "Add" button to the header

**Reply by codecatalysts** [reply]
I will agree that if the scroll-reset didn't happen this would be less of an issue.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
